src/Configuration.c

```diff
@@ -1,8 +1,68 @@
 #include "Configuration.h"
-
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
 
 struct Configuration configuration;
 
+static int wal_stricmp(const char *a, const char *b) {
+  int ca, cb;
+  do {
+     ca = (unsigned char) *a++;
+     cb = (unsigned char) *b++;
+     ca = tolower(toupper(ca));
+     cb = tolower(toupper(cb));
+   } while (ca == cb && ca != '\0');
```

Comment:
Not clear. Why it's required to use both "toupper" and "tolower"?

Reply:
This boils down to this conversation: https://stackoverflow.com/a/51992138. So in our case its mostly due to good practice. Most if not all our strings would also work with just one of them.